package.json

```diff
@@ -20,7 +20,8 @@
     "npm:tag:beta": "npm run npm:tag -- beta",
     "npm:tag:latest": "npm run npm:tag -- latest",
     "notify:docs": "node ./scripts/notify-docs/published-headless.js",
-    "doc:generate": "lerna run doc:generate"
+    "doc:generate": "lerna run doc:generate",
+    "lockLernaDependencies": "lerna exec \"npm install --package-lock-only\""
```

Comment:
`snyk monitor --all-projects` was failing because lerna dependencies (those that are symlinked) were missing from `package-lock.json` files. Even installing dependencies from scratch, lerna appears to modify lock files in some ways and omits symlinked dependencies.

This new script adds them to lock files. It also appears to replace tabs with spaces.